ci/ruby.yml

```diff
@@ -24,10 +24,7 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Set up Ruby
-    # To automatically get bug fixes and new Ruby versions for ruby/setup-ruby,
-    # change this to (see https://github.com/ruby/setup-ruby#versioning):
-    # uses: ruby/setup-ruby@v1
-      uses: ruby/setup-ruby@473e4d8fe5dd94ee328fdfca9f8c9c7afc9dae5e
+      uses: ruby/setup-ruby@v1
       with:
```

Comment:
9312141228

Reply:
I would do that and pay for it if you could give me some how to get number 9312141228File attached and everything without them knowing I will pay for that like still￼ For two days now and I still have not satisfy my phone nothing just slowing  my phone￼ don and no info